dotCMS/src/main/java/com/dotcms/rest/api/v1/system/websocket/SystemEventsWebSocketEndPoint.java

```diff
@@ -152,6 +145,26 @@ private SystemEvent processEvent(final Session session,
 		return null != processor? processor.process(event, session): event;
 	} // processEvent.
 
+    /**
+     * Verifies if the current user has the "visibility" rights to use this given payload
+     *
+     * @param session Session wrapper needed in order to obtain the current user information
+     * @param payload Payload to validate
+     * @return true the current user has "visibility" rights on this payload
+     */
+    private boolean validPayload(final SessionWrapper session,
+                                 final Payload payload) {
+
+        //Get the verifier associated to this Payload
+        final PayloadVerifier verifier = this.payloadVerifierFactory.getVerifier(payload);
+        if ( null != verifier ) {
```

Comment:
I would change this for a 

return (null != verifier)? verifier.verified(payload, session): true;